Mathlib/Probability/Moments/SubGaussian.lean

```diff
@@ -386,6 +515,123 @@ lemma measure_ge_le (h : HasSubgaussianMGF X c μ) {ε : ℝ} (hε : 0 ≤ ε) :
 
 end ChernoffBound
 
+section Add
+
+lemma add_of_indepFun {Y : Ω → ℝ} {cX cY : ℝ≥0} (hX : HasSubgaussianMGF X cX μ)
+    (hY : HasSubgaussianMGF Y cY μ) (hindep : IndepFun X Y μ) :
+    HasSubgaussianMGF (fun ω ↦ X ω + Y ω) (cX + cY) μ where
+  integrable_exp_mul t := by
+    simp_rw [mul_add, exp_add]
+    convert MemLp.integrable_mul (hX.memLp_exp_mul t 2) (hY.memLp_exp_mul t 2)
+    norm_cast
+    infer_instance
+  mgf_le t := by
+    calc mgf (X + Y) μ t
+    _ = mgf X μ t * mgf Y μ t :=
+      hindep.mgf_add (hX.integrable_exp_mul t).1 (hY.integrable_exp_mul t).1
+    _ ≤ exp (cX * t ^ 2 / 2) * exp (cY * t ^ 2 / 2) := by
+      gcongr
+      · exact mgf_nonneg
+      · exact hX.mgf_le t
+      · exact hY.mgf_le t
+    _ = exp ((cX + cY) * t ^ 2 / 2) := by rw [← exp_add]; congr; ring
+
+lemma sum_of_iIndepFun {ι : Type*} [IsZeroOrProbabilityMeasure μ]
+    {X : ι → Ω → ℝ} (h_indep : iIndepFun X μ) {c : ι → ℝ≥0}
+    (h_meas : ∀ i, Measurable (X i))
```

Comment:
This shouldn't stop the PR, but I don't see how measurability can be relevant since you already have ae measurability from `h_subG`.

Reply:
Assumption removed in #23070